src/Features/CSharp/Portable/Structure/Providers/AccessorDeclarationStructureProvider.cs

```diff
@@ -29,9 +29,13 @@ internal class AccessorDeclarationStructureProvider : AbstractSyntaxNodeStructur
                 return;
             }
 
+            SyntaxNodeOrToken current = accessorDeclaration;
+            var nextSibling = current.GetNextSibling();
+
             spans.AddIfNotNull(CSharpStructureHelpers.CreateBlockSpan(
                 accessorDeclaration,
                 accessorDeclaration.Keyword,
+                compressEmptyLines: !nextSibling.IsNode || nextSibling.AsNode() is AccessorDeclarationSyntax,
```

Comment:
why the `!nextSibling.IsNode`. might be worth a comment on these cases to just explain why that is the right thing to do.

Reply:
Collapses the blank line after the method but before the brace in this:

```csharp
class C
{
  void Method()
  {
  }

}
```

Reply:
Ah.  Seems reasonable.  in that case it might make more sense to do something more like `nextSibling == default` to indicate that there's no next sibling.  I read the above more like "i have a sibling, but it's not a node" and was scratching my head :)

Reply:
`nextSibling` could be the `CloseBraceToken` of the parent, which is not a node and also is not `default`.

Reply:
Ah.  That makes sense and isn't how i thought GetNextSibling worked.  Specifically, i thought it woudl only return the next node in it's list.  If it returns the next node/token in the parent, then that makes much more sense.

Note: nextSibling.IsToken would likely be a smidge clearer (unless you do want ot handle the 'null' case).  But i'm totally fine with this if you think this is the clearest or most accurate.